### PR TITLE
Add Linting and Formatting to the Swift SDK

### PR DESCRIPTION
**Pull request by ndurell**
Using pre-commit to run linting and formatting on every pull and on master.

### CODE REVIEW

File: Tests/KlaviyoSwiftTests/Vendor/ComposableArchitecture/TestStore.swift [hunk 1]
```diff
@@ -20,7 +20,7 @@
  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
```
Comment (ajaysubra) [inline Tests/KlaviyoSwiftTests/Vendor/ComposableArchitecture/TestStore.swift:20]: should we exclude vendor code?

File: Sources/KlaviyoSwift/Vendor/ReachabilitySwift.swift [hunk 2]
```diff
@@ -40,10 +40,10 @@ let ReachabilityChangedNotification = NSNotification.Name("ReachabilityChangedNo
 func callback(reachability:SCNetworkReachability, flags: SCNetworkReachabilityFlags, info: UnsafeMutableRawPointer?) {
```
Comment (ajaysubra) [inline Sources/KlaviyoSwift/Vendor/ReachabilitySwift.swift:40]: same question about vendor code here as well.

**Review by ajaysubra — APPROVED** [review]
👍🏽 with a question about vendor code.

File: Tests/KlaviyoSwiftTests/Vendor/ComposableArchitecture/TestStore.swift (hunk 1 above)
Comment (ndurell) [reply Tests/KlaviyoSwiftTests/Vendor/ComposableArchitecture/TestStore.swift:20]: I did for most things. I can exclude here too but seemed harmless for now.

**Pull request merged**